index.html

```diff
@@ -3078,25 +3078,62 @@ <h3>Extensibility</h3>
         <section>
           <h4>Semantic Interoperability</h4>
 
+          <p>
+            When defining new terms in an application-specific vocabulary, developers MUST use globally unambiguous [=URLs=] to identify the terms.
+            Whenever possible, it is RECOMMENDED to re-use terms — and their corresponding URLs — defined by well-known, public vocabularies,
+            such as [[[?DCTERMS]]] [[?DCTERMS]] or [[[?schema-org]]] [[?schema-org]]. 
+            If that is not possible, developers MUST <em>define</em> a new URL for each term. 
+            When doing so, general [[LINKED-DATA]] principles must be followed, in particular:
```

Comment:
Is this a normative MUST? If not, we should say something like: "are expected to be followed"

Reply:
Yeah, I did not want to put a _MUST_ here, the ld principles are fairly long. I changed it to the version you proposed.